src/views/EmotePage/EmoteComment.vue

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable prettier/prettier */
+<template>
+	<div class="comment">
+		<UserTag scale="1em"></UserTag>
+		<div class="comment-text">
+			Anyways uhm... I bought a whole bunch of shungite, rocks, do you know what shungite is? Anybody know what
+			shungite is. No, no Suge Knight, I think hes locked up in prison. Talking shungite. Anyways, its a 2 billion
+			year old like rock, stone that protects against frequencies and unwanted frequencies that may be traveling
+			in the air. So thats my story. I bought a whole bunch of stuff, put them around the la casa. Little
+			pyramids. Stuff like that.
+		</div>
+		<div class="time">03.09.2018, 19:02</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import UserTag from "@/components/utility/UserTag.vue";
+
+export default defineComponent({
+	components: {
+		UserTag,
+	},
+});
+</script>
+
+<style lang="scss" scoped>
+@import "@scss/emotes.scss";
+@import "@scss/components/user-tag.scss";
```

Comment:
Why was the the UserTag component's styling imported here?

Reply:
user-tag:
Cause it uses UserTag in comments, but we can't really import the data for it to really check. For now it's hardcoded along with CSS, but when we have data, we will use the component.

emotes:
I'm actually not sure if you have to import it for the inner component separately. Does the child component use the parent's css? If that's the case, then yea, my bad.

Reply:
Ok, tested, that is the case. I don't need those lines. Thanks!